src/components/[guild]/activity/ActivityLogFiltersBar/components/FilterTag/components/RoleSuggestions.tsx

```diff
@@ -6,12 +6,17 @@ import Suggestion from "../../Suggestion"
 import NoResults from "./NoResults"
 
 type Props = {
+  guildId?: string | number
   inputValue?: string
   getOptionProps: (props: combobox.OptionProps) => HTMLAttributes<HTMLElement>
 }
 
-const RoleSuggestions = ({ inputValue, getOptionProps }: Props): JSX.Element => {
-  const { id: guildId, roles } = useGuild()
+const RoleSuggestions = ({
+  guildId,
```

Comment:
Do we actually need to pass the `guildId` prop here (and in the `RewardSuggestions` component too), or could we use it from context (e.g. active filters context)?

Reply:
Currenlty, both of these components are used only in the `FilterTag` component, which can directly pass them the `guildId` as a prop. Why would context be preferable in this case? Is there a convention that I should follow when deliberating the usage of props over context?